lib/schema.js

```diff
@@ -324,7 +326,7 @@ function PrimitiveSchema(type) {
 
 util.inherits(PrimitiveSchema, Schema);
 
-function FieldSchema(name, type) {
+function FieldSchema(name, type, defaultValue) {
```

Comment:
I attempted to remove this argument and the `defaultValue` below and I got an AvroValidationError on the unit test for defaults. It seems like the default is actually necessary here, in particular when serializing records... am I missing something?